File: PHP/Contoh Fungsi/Fungsi (Functions) Tukar.php
<!DOCTYPE HTML>
<html>

<head>
	<title>Contoh Fungsi Tukar</title>
</head>

<body>

	<?php
	function tukar($a, $b)
	{
		$temp = $a;
		$a = $b;
		$b = $temp;

		echo "a = " . $a . "<br>";
		echo "b = " . $b . "<br>";
	}
	?>

	<?php
	$a = 1;
	$b = 2;

	tukar($a, $b)
	?>

</body>

</html>

File: Kotlin/Contoh Fungsi/Fungsi (Functions) Tukar.kt
fun tukar(a: Int, b: Int) {
        var temp: Int
        temp = a
        var a = b
        var b = temp
        println("a = " + a)
        println("b = " + b)
}

fun main(args: Array<String>) {
        var a = 1
        var b = 2
        tukar(a, b)
}
